### PR TITLE
Fixing duplicated port for profile downloading.

### LINKED ISSUE #2718: Profile download for iOS not possible
**Describe the bug, try to make it reproducible**
I changed the ports in mailcow.conf to 8081 (HTTP) and 1443 (HTTPS). When I am trying  to download the Profile following url is called:

`https://(replaced domain):1443:1443/mobileconfig.php`

**System information:**

 - Ubuntu 18.04 LTS
 - Proxmox
 - 30 GB Ram 8 cores
 - Docker version 18.09.6, build 481bc77
 - docker-compose version 1.24.0, build 0aa59064

### PR DESCRIPTION
**Pull request by mkapra**
Fixes #2718

### CODE REVIEW

File: data/web/user.php [hunk 1]
```diff
@@ -76,7 +76,10 @@
   $_SESSION['return_to'] = $_SERVER['REQUEST_URI'];
   $username = $_SESSION['mailcow_cc_username'];
   $mailboxdata = mailbox('get', 'mailbox_details', $username);
-
+	
+  $mailcow_hostname_array = explode(":", $mailcow_hostname)
+  $mailcow_hostname = $mailcow_hostname_array[0]
+
```
Comment (mkuron) [inline data/web/user.php]: That would work, but change it to use the same `strtok` solution that we have in the line below.

**Review by mkuron — CHANGES_REQUESTED** [review]

File: data/web/user.php (hunk 1 above)
Comment (patschi) [reply data/web/user.php]: This wouldn't work, to be correct. The missing semicolons `;` would cause a parse error.

Comment (mkapra) [reply data/web/user.php]: @patschi  ah sure! sorry I was in python :D

**Comment by mkapra** [comment]
@mkuron 
I changed the explode to strtok() and rebased the branch so there is only one commit.
Is it okay now? :)

**Comment by mkuron** [comment]
Looks good. Did you test whether it solves your problem?

**Comment by mkapra** [comment]
> Looks good. Did you test whether it solves your problem?

@mkuron How can I use my branch instead of another on my mailserver or do I don't understand you?

**Comment by mkuron** [comment]
Go to your mailcow-dockerized directory and run the following commands:

```
git add remote mkapra https://github.com/mkapra/mailcow-dockerized.git
git fetch mkapra
git checkout mkapra/patch-1
```

**Comment by mkuron** [comment]
And actually your patch won't work. You need to do `strtok` before the `urlencode`. Same goes for the HTTP_HOST. Could you fix it there too?

**Comment by mkapra** [comment]
@mkuron: I fixed your suggestions. But when I try to execute:
`git remote add mkapra https://github.com/mkapra/mailcow-dockerized.git`

I am getting the following error:
`fatal: pathspec 'remote' did not match any files`

Do you know what I am doing wrong?

**Comment by mkapra** [comment]
@mkuron Found the problem. I was only typing the command and not verifying it.
It should be called like this:
`git remote add ...`

**Comment by mkapra** [comment]
@mkuron 
I can  confirm that my fix works! Rebased it again.

**Comment by Braintelligence** [comment]
@mkuron Are you 100% sure about the strtok order? The urlencode was what returned a urlencoded string with a colon that had to be stripped, why should the colon be stripped from the parameter of urlencode instead?

**Comment by mkapra** [comment]
> @mkuron Are you 100% sure about the strtok order? The urlencode was what returned a urlencoded string with a colon that had to be stripped, why should the colon be stripped from the parameter of urlencode instead?

@Braintelligence 
I am cutting at the ":" sign. After url encode the ":" would be "&#58" so the strtok will not work. I also checked my fix already and it works.

**Comment by Braintelligence** [comment]
My earlier PR should also be changed then. I'll create a new PR in a few mins.

**Comment by mkapra** [comment]
@Braintelligence I also can fix this if you want?

**Comment by mkapra** [comment]
@Braintelligence I see that I already fixed this in this pull request by demand of @mkuron. So no need to change anything.

**Comment by Braintelligence** [comment]
Oh it was the same line as my earlier PR?

**Comment by mkapra** [comment]
@Braintelligence Yes. The problem was that the port was also in the host option. I am thinking about if we need the strtok at HTTP_PORT? I think it's already right there.
I will test it without it.

**Comment by Braintelligence** [comment]
Yup, you touched a new part and also the old one with the wrong strtok order. I approve :)

**Comment by mkapra** [comment]
@Braintelligence @mkuron I was wrong by adding strtok() to the host option. @Braintelligence 's fix was nearly the right one. It only was the order of urlencode() and strtok() there.
I tried it already on my server.

**Comment by andryyy** [comment]
Looks good to me now. Any complains?

**Review by mkuron — APPROVED** [review]
Haven't tested it, but it looks correct to me now.

**Review by Braintelligence — APPROVED** [review]

**Pull request merged**